### ISSUE TITLE
Change colour of assistive technology focus border

### ISSUE THREAD

**Issue by LucyGwilliamAdmin** [open-sdg/open-sdg#849]
**Page 120 in DAC report**

**Issue from DAC report:**
When tabbing through page with assistive technology, components are highlighted with black border to indicate focus. The black focus highlight is not visible against the black ‘high contrast’ button.

**Solution from DAC report:**
Please ensure that a mechanism is in place to allow users to identify their focus on the page. In this case, the black border is not visible against the black button. The GDS focus highlight is recommended. Consider implementing a consistent visible focus indicator to help users identify when page elements are in focus, using the ‘Focus state’ colour ($govukfocus- colour #FFDD00). For more information please visit [Colour - GOV.UK Design System](https://design-system.service.gov.uk/styles/colour/), Focus state.

**Example from DAC report:**
No example provided
